src/components/FormikAutocomplete.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { InputLabel, TextField } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import PropTypes from 'prop-types';
+
+/* eslint-disable react/jsx-props-no-spreading */
+function FormikAutocomplete(props) {
```

Comment:
It'd be nice if this accepted a placeholder value that indicates any/all values apply. E.g. For the Majors one it could say "All/Any majors acceptable" or "No major requirements"

Reply:
added!